src/net.cpp

```diff
@@ -1919,7 +1922,7 @@ std::vector<CAddress> CConnman::GetCurrentBlockRelayOnlyConns() const
     return ret;
 }
 
-std::vector<AddedNodeInfo> CConnman::GetAddedNodeInfo() const
+std::vector<AddedNodeInfo> CConnman::GetAddedNodeInfo(bool includeConnected) const
```

Comment:
style (same in the declaration):
```suggestion
std::vector<AddedNodeInfo> CConnman::GetAddedNodeInfo(bool include_connected) const
```